### DISCUSSION TITLE
Retransmis

### DISCUSSION THREAD

**Discussion by Crisatian** [helium/miner#1656] [Issues]
I have the host sent because of your abdeits, I'm sorry to tell you, but I have all the ports open on my router + public IP which does not change if I have the current problem with the internet for a few minutes and then returns to normal, and hundred all transmitted, I know they are not a prioritycompared to the problems you have with the network, but solve my problem because it is not from me, it appeared on 11.05. 2022 with the new abdication.

**Comment by meowshka** [comment]
@Crisatian what miner do you have? Can you please share link to your hotspot on the Explorer?
